sdk/keyvault/keyvault-secrets/review/keyvault-secrets.api.md

```diff
@@ -68,7 +68,9 @@ export { PollOperationState }
 export interface PurgeDeletedSecretOptions extends coreHttp.OperationOptions {
 }
 
-// @public
+// Warning: (ae-internal-missing-underscore) The name "RecoverDeletedSecretOptions" should be prefixed with an underscore because the declaration is marked as @internal
+// 
+// @internal
 export interface RecoverDeletedSecretOptions extends coreHttp.OperationOptions {
```

Comment:
`RecoverDeletedSecretOptions ` should not be exported

Reply:
Moved to: https://github.com/Azure/azure-sdk-for-js/issues/5907#issuecomment-547576995